Onboarding/Sources/Onboarding/Components/Views/UIKitPageController.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import UIKit
+import SwiftUI
+
+struct UIKitPageControl: UIViewRepresentable {
+    @Binding var currentPage: Int
+    var numberOfPages: Int
+
+    func makeUIView(context: Context) -> UIPageControl {
+        let pageControl = UIPageControl()
+        pageControl.addTarget(
+            context.coordinator,
+            action: #selector(Coordinator.valueChanged(_:)),
+            for: .valueChanged
+        )
+        pageControl.currentPage = currentPage
+        pageControl.isAccessibilityElement = false
+        pageControl.numberOfPages = numberOfPages
+        pageControl.currentPageIndicatorTintColor = UIColor(
+            Color("AccentColor", bundle: Bundle.module))
+         DispatchQueue.main.async {
```

Comment:
Does this need to be here?

Reply:
This needs to be here if you take it out, an error will appear  when the pageViewController is changed

Reply:
I'm not sure it does.

Reply:
removed